src/ui/AdaptyOnboardingView.tsx

```diff
@@ -2,17 +2,21 @@ import React, { memo, useEffect, useMemo } from 'react';
 import { requireNativeComponent, ViewProps } from 'react-native';
 import { AdaptyOnboarding } from '@/types';
 import { AdaptyOnboardingCoder } from '@/coders/adapty-onboarding';
+import { AdaptyUICreateOnboardingViewParamsCoder } from '@/coders';
 import { generateId } from '@/utils/generate-id';
 import { shouldEnableMock } from '@/utils';
 import {
+  CreateOnboardingViewParamsInput,
   OnboardingEventHandlers,
   NativeAdaptyOnboardingViewProps,
 } from './types';
 import { createOnboardingEventHandlers } from './create-onboarding-event-handlers';
+import { DEFAULT_ONBOARDING_PARAMS } from './onboarding-view-controller';
 import { AdaptyOnboardingViewMock } from './AdaptyOnboardingView.mock';
 
 export type Props = ViewProps & {
   onboarding: AdaptyOnboarding;
+  params?: CreateOnboardingViewParamsInput;
```

Comment:
params is like deprecated eventHandlers. let's add props externalUrlsPresentation